libreyolo/models/base/model.py

```
@@ -921,15 +921,17 @@ def track(
         show: bool = False,
         vid_stride: int = 1,
         output_path: Optional[str] = None,
+        tracker: str = "bytetrack",
         tracker_config=None,
         augment: bool = False,
```

Ruff (0.15.15)

[warning] Unused method argument: `augment`

(ARG002)